packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/layout/layout.ts

```diff
@@ -115,8 +118,21 @@ export const prepareLayoutArea = (diagram: Diagram, renderCallback: () => void):
   const diagramDirectEditContextProvider = createElement(DiagramDirectEditContextProvider, {
     children: hiddenContainerContentElements,
   });
+  const reactFlowContextProvider = createElement(ReactFlowProvider, { children: diagramDirectEditContextProvider });
+  const toastContextProvider = createElement(ToastContext.Provider, {
+    value: {
+      useToast: () => {
+        return useSnackbar();
+      },
+    },
+    children: reactFlowContextProvider,
+  });
+  const cache = new InMemoryCache();
   ReactDOM.render(
-    createElement(ReactFlowProvider, { children: diagramDirectEditContextProvider }),
+    createElement(ApolloProvider, {
+      client: new ApolloClient({ cache: cache, uri: '' }),
+      children: toastContextProvider,
+    }),
     hiddenContainer,
```

Comment:
` 
const reactFlowContextProvider = createElement(ReactFlowProvider, { children: diagramDirectEditContextProvider });
  const toastContextProvider = createElement(ToastContext.Provider, {
    value: {
      useToast: () => {
        return useSnackbar();
      },
    },
    children: reactFlowContextProvider,
  });
  const cache = new InMemoryCache();
  ReactDOM.render(
    createElement(ApolloProvider, {
      client: new ApolloClient({ cache: cache, uri: '' }),
      children: toastContextProvider,
    }),
    `
    @sbegaudeau @gcoutable 
    I had to recreate all the contexts used by ListNode and useDrop, I'm not sure about that.

Reply:
@sbegaudeau @gcoutable

Reply:
We will discuss that after the next release